src/main/java/com/icthh/xm/ms/otp/config/Constants.java

```diff
@@ -12,7 +12,10 @@
     public static final String LOGIN_REGEX = "^[_.@A-Za-z0-9-]*$";
 
     public static final String SYSTEM_ACCOUNT = "system";
+    public static final String X_TENANT = "x-tenant";
+    public static final String ACCESS_TOKEN = "access_token";
     public static final String ANONYMOUS_USER = "anonymoususer";
+    public static final String MSISDN = "msisdn";
```

Comment:
Better to use a more universal name, like  `receiver` - in future, it also could be an email, IP, etc.